build-farm/platform-specific-configurations/mac.sh

```diff
@@ -53,14 +54,14 @@ if [ "${VARIANT}" == "openj9" ]; then
   export PATH=/usr/local/bin:$PATH
   # ccache causes too many errors (either the default version on 3.2.4) so disabling
   export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-ccache"
+  export MACOSX_DEPLOYMENT_TARGET=10.9.0
   if [ "${JAVA_TO_BUILD}" == "${JDK8_VERSION}" ]
   then
     export SED=gsed
     export TAR=gtar
     export UMA_SUPPRESS_WARNINGS_AS_ERRORS=1
     export OMR_WARNINGS_AS_ERRORS=0
```

Comment:
Referring to https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/561#issuecomment-438809076: we don't need to set `UMA_SUPPRESS_WARNINGS_AS_ERRORS` and `OMR_WARNINGS_AS_ERRORS` anymore.